delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/remote/client/RemoteDiskStorageClient.scala

```diff
@@ -107,7 +107,7 @@ final class RemoteDiskStorageClient(client: HttpClient, getAuthToken: AuthTokenP
     *   the relative path to the file location
     */
   def getFile(bucket: Label, relativePath: Path)(implicit baseUri: BaseUri): IO[FetchFileRejection, AkkaSource] = {
-    getAuthToken(credentials).toBIO.flatMap { authToken =>
+    getAuthToken(credentials).toBIOUnsafe.flatMap { authToken =>
```

Comment:
`toBioThrowable`? It's not unsafe in this instance. I think that name is nice because it explains why it would be unsafe

Reply:
Haha fair, unsafe tends to be used to mean "think twice about what you're about to do", which is what I was going for 😂

Reply:
Although I'm not sure I would understand that `toBioThrowable` was unsafe without either knowing CE and monix well or by looking at the method comment

Reply:
Ahh I guess the real issue is that often we hide the error channel by using `UIO` or some other type alias. Think that's why it wasn't obvious from the places I looked